site/en/docs/lighthouse/overview/index.md

```diff
@@ -7,13 +7,15 @@ date: 2016-09-27
 updated: 2022-05-24
 ---
 
-[Lighthouse](https://github.com/GoogleChrome/lighthouse) is an open-source, automated tool for improving the quality of web pages. You can run it against any web page, public or requiring authentication. It has audits for performance, accessibility, progressive web apps, SEO and more.
+[Lighthouse](https://github.com/GoogleChrome/lighthouse) is an open-source, automated tool for improving the quality of web pages. You can run it against any web page, public or requiring authentication. It has audits for performance, accessibility, progressive web apps, SEO, and more.
 
 <figure>
   {% Img src="image/MtjnObpuceYe3ijODN3a79WrxLU2/QXJ5lmcjHEFLTHg5B4o8.svg", class="float-right", alt="Lighthouse logo", width="150", height="150" %}
 </figure>
 
-You can run Lighthouse in Chrome DevTools, from the command line, or as a Node module. You give Lighthouse a URL to audit, it runs a series of audits against the page, and then it generates a report on how well the page did. From there, use the failing audits as indicators on how to improve the page. Each audit has a reference doc explaining why the audit is important, as well as how to fix it.
+You can run Lighthouse in Chrome DevTools, from the command line, or as a Node module. You give Lighthouse a URL to audit, it runs a series of audits against the page,
+and then it generates a report on how well the page did. From there, use the failing audits as indicators on how to improve the page. Each audit has a reference doc explaining why
+the audit is important, as well as how to fix it.
```

Comment:
Looks ugly here splitting this line makes the page more navigable: 

[https://raw.githubusercontent.com/GoogleChrome/developer.chrome.com/f4eb081d346113aa2baa92628108c8baa8aba90b/site/en/docs/lighthouse/overview/index.md](https://raw.githubusercontent.com/GoogleChrome/developer.chrome.com/f4eb081d346113aa2baa92628108c8baa8aba90b/site/en/docs/lighthouse/overview/index.md)